### PR TITLE
Subscribe users to the newsletter after they signup for a membership

### PR DESCRIPTION
**Pull request by orlando**
**What:** Subscribe users to the newsletter after they signup for a membership

**Why:** Related to https://app.asana.com/0/1168997577035609/1188545238323753

**How:**

- Add gibbon MailChimp gem
- Add background job to subscribe users to a mailchimp newsletter

**To do:**
- [x] trigger newsletter job after user signups
- [x] add Mailchimp tags giving the status of the user membership

### CODE REVIEW

**Review by arathjz — APPROVED** [review]

**Comment by orlando** [comment]
This is ready for review. I've just tested this locally and works good. Also, tests included

File: app/models/user.rb [hunk 1]
```diff
@@ -92,6 +92,23 @@ def admin?
     !!admin
   end
 
+  # TODO: We are getting first_name and last_name from users when the join the union
```
Comment (duranmla) [inline app/models/user.rb:95]: Do we have a ticket created for this?
_Make sure to keep this tech debt visible..._

**Review by duranmla — APPROVED** [review]
Great! Thanks!

File: app/models/user.rb (hunk 1 above)
Comment (orlando) [reply app/models/user.rb:95]: https://app.asana.com/0/1195530778765030/1199544593172957

**Pull request merged**